eo-runtime/src/main/java/org/eolang/Universe.java

```diff
@@ -122,20 +132,30 @@ public int copy(final int vertex) {
      */
     public byte[] dataize(final int vertex) {
         return new Param(
-            Optional.ofNullable(
-                this.indexed.get(vertex)
-            ).orElseThrow(
-                () -> new ExFailure(
-                    String.format(
-                        "Phi object with vertex %d was not indexed.",
-                        vertex
-                    )
-                )
-            ),
+            this.get(vertex),
             "Δ"
         ).asBytes().take();
     }
 
+    /**
+     * Find phi by vertex.
+     * @param vertex Vertex.
+     * @return Phi.
+     * @throws ExFailure if vertex does not exist in the map.
+     */
+    private Phi get(final int vertex) {
+        return Optional.ofNullable(
+            this.indexed.get(vertex)
+        ).orElseThrow(
+            () -> new ExFailure(
+                String.format(
+                    "Phi object with vertex %d was not indexed.",
+                    vertex
```

Comment:
@levBagryansky Same here. Can we provide some more details to the error message?

Reply:
@Graur Seems no, it just throws exception if the vertex is absent in the map